src/jit/codegenlinear.h

```diff
@@ -116,7 +116,7 @@ void genPutArgStkSIMD12(GenTree* treeNode);
 
 #if FEATURE_HW_INTRINSICS && defined(_TARGET_XARCH_)
 void genHWIntrinsic(GenTreeHWIntrinsic* node);
-void genHWIntrinsic_R_R_RM(GenTreeHWIntrinsic* node, instruction ins);
+void genHWIntrinsic_R_R_RM(GenTreeHWIntrinsic* node, instruction ins, HWIntrinsicCategory category);
```

Comment:
What is the category field needed for here?

Reply:
I added a `Commutative` category that we may need it to fold op1.

Reply:
I think we can handle that by swapping op1 and op2 in lowering so we don't need to handle it later.